plugin-maven/README.md

```diff
@@ -115,6 +115,60 @@ By default, all files matching `src/main/java/**/*.java` and `src/test/java/**/*
 </configuration>
 ```
 
+<a name="format"></a>
+
+## Applying to custom sources
+
+By default, no Ant-Style include patterns are defined.  Each element under `<format>` is a step, and they will be applied in the order specified.  Every step is optional, and they will be applied in the order specified.
+
+```xml
+<configuration>
+  <format>
+    <includes>
+      <!-- include all property files in "resource" folders under "src" -->
+      <include>src/**/resources/**/*.properties</include>
+    </includes>
+
+    <licenseHeader>
+      <!-- Specify either content or file, but not both -->
+      <content>/* Licensed under Apache-2.0 */</content>
+      <file>${basedir}/license-header</file>
+      <!-- conent until first occurrence of the delimiter regex will be interpreted as header section -->
+      <delimiter>#</delimiter>
+    </licenseHeader>
+
+    <!-- Files must end with a newline -->
+    <endWithNewline />
+
+    <!-- Specify whether to use tabs or spaces for indentation -->
+    <indent>
+      <!-- Specify either withSpaces or withTabs. Defaults to withSpaces -->
+      <withSpaces>true</withSpaces>
```

Comment:
Maybe `<spaces>` instead of `<withSpaces>` and `<tabs>` instead of `<withTabs>`?

Reply:
I have thought about that too. The idea was to provide better readability (indent > withSpaces), but I guess thats not common for maven plugins. It's more of the Groovy DSL style.

Reply:
fixed